### PR TITLE
2022

### PR DESCRIPTION
**Pull request by arfrie22**
Moved to wpilib 2022

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -9,71 +11,79 @@ targetCompatibility = JavaVersion.VERSION_11
...
 }
 
+def deployArtifact = deploy.targets.roborio.artifacts.frcJava
+
+// Set to true to use debug for JNI.
+wpi.java.debugJni = false
+
 // Set this to true to enable desktop support.
-def includeDesktopSupport = true
+def includeDesktopSupport = false
```
Comment (john-petrangelo) [inline build.gradle:48]: I'm not sure you want this change. This is needed to use the simulator (and Romi).

**Review by john-petrangelo — APPROVED** [review]
lgtm, have you tested the changes on one of the robots? Don't merge until you prove the changes with at least one robot.

File: build.gradle (hunk 1 above)
Comment (arfrie22) [reply build.gradle:48]: Yeah I didn’t mean to do this, I can fix that before I merge

**Comment by arfrie22** [comment]
This works on 2019 which is running the new rio image

**Pull request merged**